### PR TITLE
WIP: Split up acceptance tests again.

### PR DESCRIPTION
**Pull request by billyvg**
We had a problem where failed builds couldn't be retried, since percy
considered the build finalized. This changes the percy build to only
finalize if the test run was successful. This change probably means we
don't get screenshots on failed tests, not sure if that's something people
care about?

We also use build id + build run so that if someone re-runs all builds then
they'll successfully get new snapshots.

### CODE REVIEW

**Comment by scefali** [comment]
@billyvg If this works, can we use the same Percy token for the plugin acceptance tests? We are currently passing in the token for the Plugin repo so they don't actually show up in Percy for the right project.

**Comment by billyvg** [comment]
Yes, but like dan suggested [here](https://github.com/getsentry/sentry/pull/12705#discussion_r340781459) , we can face problems if builds don't start within ~1 minute of each other

File: Makefile [hunk 1]
```diff
@@ -176,6 +176,9 @@ test-symbolicator:
 	@echo ""
 
 test-acceptance: node-version-check
+	echo $(PERCY_PARALLEL_NONCE)
+	echo $(PERCY_PARALLEL_NONCE2)
+	echo $(PERCY_PARALLEL_NONCE3)
```
Comment (markstory) [inline Makefile]: If you have 2 groups will you have 3 nonces?

Comment (billyvg) [reply Makefile]: @markstory I was just debugging with this PR, but will bring this conversation back to the original PR.

**Pull request closed without merging**